src/pudl/settings.py

```diff
@@ -508,7 +525,9 @@ class Ferc6XbrlToSqliteSettings(FercGenericXbrlToSqliteSettings):
     """
 
     data_source: ClassVar[DataSource] = DataSource.from_id("ferc6")
-    years: list[int] = data_source.working_partitions["years"]
+    years: list[int] = [
+        year for year in data_source.working_partitions["years"] if year >= 2021
+    ]
```

Comment:
As I mentioned with the FERC 2, I think it probably makes sense to use a validator in this class to ensure that only appropriate years can be requested.